tvtk/pyface/tvtk_scene.py

```diff
@@ -808,7 +817,7 @@ def _create_control(self, parent):
 
         self._interactor.initialize()
         self._interactor.render()
-        self.light_manager = light_manager.LightManager(self)
+        self.light_manager = light_manager.LightManager(weakref.proxy(self))
```

Comment:
similar comment https://github.com/enthought/mayavi/pull/212/files#r34133394. Could `LightManager` handle the weak proxy ref internarly?